Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -521,10 +521,19 @@ quoteListTopSheetView state push =
           ]
           []
         ]
-
 noQuotesErrorModel :: forall w . QuoteListModelState -> PrestoDOM (Effect Unit) w
 noQuotesErrorModel state =
-  linearLayout
+  let city = getCityFromString $ getValueToLocalStore CUSTOMER_LOCATION
+      imageString = if state.vehicleVariant == "AUTO_RICKSHAW" 
+                    then case city of
+                      Bangalore -> "ny_ic_no_quotes_auto_bang_del"
+                      Kochi -> "ny_ic_no_quotes_auto_koc"
+                      Delhi ->"ny_ic_no_quotes_auto_bang_del"
+                      Hyderabad -> "ny_ic_no_quotes_auto_che_hyd"
+                      Chennai -> "ny_ic_no_quotes_auto_che_hyd"
+                      _ -> "ny_ic_no_quotes_auto"
+                    else  "ny_ic_no_quotes_color"
+  in linearLayout
```

Comment:
function for auto icon based on city is already present in `Utils.purs` please reuse or modify that if required

Reply:
Done